docs/FrameDescriptions.md

```diff
@@ -1,21 +1,121 @@
+## Inertial Frame
+
+We define the inertial frame in the human side as following:
+
+<p align="center">
+  <img width="300" src="./images/InertialFrame.jpg">
+</p>
+
+## Oculus Device
+According to the [Oculus documentation](https://developer.oculus.com/documentation/native/pc/dg-sensor/), the frames attached to the Oculus Virtual Reality (OVR) headset are as following:
+```
+The x-z plane is aligned with the ground regardless of camera orientation.
+As seen from the diagram, the coordinate system uses the following axis definitions:
+- Y is positive in the up direction.
+- X is positive to the right.
+- Z is positive heading backwards.
+Rotation is maintained as a unit quaternion, but can also be reported in yaw-pitch-roll form. Positive rotation is counter-clockwise.
+```
+However, the custom they have used for defining the euler angles and their composition order are different from ours.
+[Following line of code](https://github.com/robotology/walking-teleoperation/blob/master/modules/Oculus_module/src/OculusModule.cpp#L386-L389
+) provides us the rotation from [inertial frame](#inertial-frame) (attached to the human head at the begning) to the oculus headset at timestep `t` expressed in preferred coordinate system:
+
+```
+iDynTree::toEigen(m_oculusRoot_T_headOculus).block(0, 0, 3, 3)
+                = iDynTree::toEigen(iDynTree::Rotation::RPY(-desiredHeadOrientationVector(1),
+                                                            desiredHeadOrientationVector(0),
+                                                            desiredHeadOrientationVector(2)));
+```                                                           
 
 <p align="center">
   <img width="500" height="500" src="./images/Oculus.png">
 </p>
 
-**Virtualizer (treadmill)**
+## treadmill (Virtualizer Cyberith)
+
+According to the measurement information we acquire from the cyberith treadmill, we identify the frame of Virtualizer as following:
 
 <p align="center">
-  <img width="600" height="300" src="./images/Virtualizer%2Bothers.jpg">
+  <img width="300" src="./images/virtualizerFrame.jpg">
 </p>
 
-**Xsens Suit**
+Threfore [we have:](https://github.com/robotology/walking-teleoperation/blob/60b449e6e8d5120a2a11ca2997521f46c51821c1/modules/Oculus_module/src/HeadRetargeting.cpp#L105)
+
+```
+m_oculusInertial_R_teleopFrame = iDynTree::Rotation::RotZ(-playerOrientation);
+```
+The rotation of the operator (called player here) inside the treadmill is defined with `playerOrientation`, and the rotation of it is expressed in preferred coordinates of the `I_treadmill`. In order to express the rotation in [inertial coordinates](#inertial-frame) (in the code is mentioned by `oculusInertial`), we perform the following transformation
+```
+ rotz(- playerOrientation)= rotx(pi) * rotz(playerOrientation) * rotx(-pi)
+```
+When the human rotates inside the virtualizer, not only he has rotation around `z` axis, but also the _teleoperaition frame_  translates as following figure shows:
```

Comment:
Note that the _teleoperation frame_ is not defined here. Also, there is a typo "teleoperaition"

Reply:
done